src/tools/rbd/action/Group.cc

```diff
@@ -254,8 +254,8 @@ int execute_remove_image(const po::variables_map &vm,
       vm, at::ARGUMENT_MODIFIER_NONE, arg_index);
 
   if (!image_id.empty() && has_image_spec) {
-    std::cerr << "rbd: trying to access image using both name and id. "
-              << std::endl;
+    std::cerr << "rbd: trying to access image using either name or id. "
+              << cpp_strerror(-EINVAL) << std::endl;
```

Comment:
Nit: no need to hard-code the errno string in the console message. We really only do that when we don't know what the error code could be.

Reply:
... this comment applies throughout.